src/celeritas/ext/VecgeomParams.cc

```diff
@@ -82,6 +83,40 @@ VecgeomParams::VecgeomParams(const std::string& filename)
         vgdml::Frontend::Load(filename, validate_xml_schema);
     }
 
+    Initialize();
+}
+
+#ifdef CELERITAS_USE_GEANT4
```

Comment:
These are defined as 0/1
```suggestion
#if CELERITAS_USE_GEANT4
```